dlt/load/load.py

```diff
@@ -371,6 +397,18 @@ def complete_package(self, load_id: str, schema: Schema, aborted: bool = False)
             f"All jobs completed, archiving package {load_id} with aborted set to {aborted}"
         )
 
+    def update_loadpackage_info(self, load_id: str) -> None:
```

Comment:
I am not quite sure when this needs to be run and how often, so any help there would be apreciated